tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/modules/ImportLiferayModuleProjectWizard.sdef

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+/*******************************************************************************
+ * Copyright (c) 2000-2014 Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ *
+ *******************************************************************************/
+ -->
+<definition>
+    <import>
+        <package>org.eclipse.sapphire.ui.renderers.swt</package>
+        <package>com.liferay.ide.project.ui.action</package>
+        <package>com.liferay.ide.project.ui.wizard</package>
+        <package>org.eclipse.sapphire.ui.forms.swt</package>
+    </import>
+    <wizard>
+        <id>ImportLiferayModuleProjectWizard</id>
+        <element-type>com.liferay.ide.project.core.modules.ImportLiferayModuleProjectOp</element-type>
+        <label>Import Liferay Module Project</label>
+        <page>
+            <id>ImportLiferayModuleProjectPage</id>
+            <label>Import Liferay Module Project</label>
+            <content>
+                <property-editor>Location</property-editor>
```

Comment:
Add a property editor for "buildtype" even though it wil be readonly and derived.